lib/graphql/schema/field.rb

```diff
@@ -48,7 +48,7 @@ def to_graphql
           GraphQL::Field.new
         end
 
-        field_defn.name = camelize(name)
```

Comment:
It looks like the implementation was moved from this class into `BuildType`, can we remove the code from this file?

Reply:
oops removed!